consensus/src/config.rs

```diff
@@ -70,6 +74,13 @@ impl Committee {
         2 * total_votes / 3 + 1
     }
 
+    pub fn random_coin_threshold(&self) -> Stake {
+        // If N = 3f + 1 + k (0 <= k < 3)
+        // then (2 N + 3) / 3 = 2f + 1 + (2k + 2)/3 = 2f + 1 + k = N - f
+        let total_votes: Stake = self.authorities.values().map(|x| x.stake).sum();
+        2 * total_votes / 3 + 1
+    }
+
```

Comment:
Yes, but later we may change it to a different threshold like n/3. I know the VABA paper uses a threshold of n/3.

Reply:
I see, so it has a different purpose than `quorum_threshold`?

Reply:
It is used to tell if enough randomness shares are collected. If we implement shared randomness with a different threshold then we may keep it, otherwise we can reuse quorum_threshold.